src/main/scala/no/gdl/bookapi/service/ValidationService.scala

```diff
@@ -20,6 +21,17 @@ trait ValidationService {
   val validationService: ValidationService
 
   class ValidationService extends LazyLogging {
+    def validateUpdatedFeaturedContent(content: api.FeaturedContent): Try[api.FeaturedContent] = {
+      // TODO Check stuff
```

Comment:
Need to discuss what we want to validate here.

Reply:
Anytime 😄